src/Nox.Lib/Presentation/Api/OData/ApiRouteQueryStringOperationFilter.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.OpenApi.Models;
+using Nox.Solution;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Nox.Presentation.Api.OData;
+
+public class ApiRouteQueryStringOperationFilter: IOperationFilter
+{
+    private readonly IEnumerable<string> _auditEntityPaths;
+
+    public ApiRouteQueryStringOperationFilter(NoxSolution noxSolution)
+    {
+        _auditEntityPaths = 
+            noxSolution.Domain!.Entities.Where(e => e.Persistence.IsAudited)
+                .Select(e=> string.Concat(noxSolution.Presentation.ApiConfiguration.ApiRoutePrefix.Trim('/'), "/", e.PluralName));
```

Comment:
ApiRoute Prefix is already sanitize. Maybe you can also consider using string interpolation
```suggestion
                .Select(e=> string.Concat(noxSolution.Presentation.ApiConfiguration.ApiRoutePrefix, "/", e.PluralName));
```

Reply:
Desired changes have been applied